__tests__/server.test.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const supertest = require('supertest');
+const { server } = require('../src/server');
+const mockRequest = supertest(server);
+const { dbInstance } = require('../src/models/index');
```

Comment:
When destructuring, your names have to match the property name of the object, in this case the names you used when you exported them.

```js
const { app } = require('../src/server');
const mockRequest = supertest(app);
const { sequelizeDatabase } = require('../src/models/index');